tasks/src/depth-estimation/data.ts

```diff
@@ -1,7 +1,12 @@
 import type { TaskDataCustom } from "../Types";
 
 const taskData: TaskDataCustom = {
-	datasets: [],
+	datasets: [
+		{
+			decription: "NYU Depth Dataset"
```

Comment:
```suggestion
			description: "NYU Depth Dataset",
```

Reply:
I feel like it would be better to add a simpler explanation to description, WDYT?

Reply:
Thanks! I updated the corrections. In addition, I added some more explanation for the dataset.